src/sql/platform/accounts/common/accountStore.ts

```diff
@@ -8,15 +8,20 @@ import { AccountAdditionResult } from 'sql/platform/accounts/common/eventTypes';
 import { IAccountStore } from 'sql/platform/accounts/common/interfaces';
 import { deepClone } from 'vs/base/common/objects';
 import { firstIndex } from 'vs/base/common/arrays';
+import { ILogService } from 'vs/platform/log/common/log';
 
 export default class AccountStore implements IAccountStore {
+	private readonly deprecatedProviders = ['azurePublicCloud'];
```

Comment:
This is what it used to look like: https://github.com/microsoft/azuredatastudio/blob/b631530753fb12e6edc4b15cff6023016cf18e88/extensions/azurecore/src/account-provider/providerSettings.ts#L17

This is what it currently looks like:
https://github.com/microsoft/azuredatastudio/blob/e333eb584da5b33e6cab0a80a701b580807936b5/extensions/azurecore/src/account-provider/providerSettings.ts#L16